blog/gag-stack.md

```
@@ -0,0 +1,149 @@
+title: "Development with AI: the GAG Stack"
+date: 2024-03-29
+tags: ["ai", "development", "tools"]
+# Development with AI: the GAG Stack
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)